### DISCUSSION TITLE
Keycloak-Quarkus  21.1.1

### DISCUSSION THREAD

**Discussion by ugam0910** [keycloak/keycloak#22566] [Archive]
Hi Team,

We are implementing Keycloak  21.1.1 (Quarkus based)  on Azure cloud which is behind nginx reverse proxy
We have created docker file based on openjdk:11-alpine , in which keycloak 21.1. is used
---------------
we have setup in docker file  following env variables as : 
ENV KEYCLOAK_ADMIN=keycloak
ENV KEYCLOAK_ADMIN_PASSWORD=keycloak
ENV KEYCLOAK_FRONTEND_URL=https://<serverhost:port/auth
.....
.....
.....
ENTRYPOINT ["/opt/keycloak-21.1.1/bin/kc.sh start-dev"]
--------------

we able to sett 'welcome' page of keycloak with url : https://serverhost:port/auth but on click on 'administration portal' , new page get opened with 'loading the Admin UI' and nothing happens
on F12, we see 'failed to load resource : the servcer responded with a status of 502 ()'

can team help us to resolve this problem

**Comment by ugam0910** [comment]
Hi team , just to add, initially we did not set 'KEYCLOAK_FRONTEND_URL' , that time we are getting 'mixed content' error , which is not gone post  'KEYCLOAK_FRONTEND_URL'

**Comment by kkcmadhu** [comment]
keycloak need  to be aware its behind reverse proxy/nginx. this section may be helpfult for you
https://www.keycloak.org/server/reverseproxy

most straight forward setup will be to  add this additonal env variable KC_PROXY=edge

**Comment by vmuzikar** [comment]
@ugam0910 Can you please report this under https://github.com/keycloak/keycloak/issues/14666?
